packages/entity-browser/src/modules/searchForm/sagas.js

```diff
@@ -27,6 +27,7 @@ export function* initialize({payload}) {
 
   const relationEntities = yield formDefinition.filter(searchField =>
     searchField.type === 'ch.tocco.nice2.model.form.components.simple.MultiSelectBox'
+    || searchField.type === 'ch.tocco.nice2.model.form.components.simple.SingleSelectBox'
```

Comment:
The babel polyfill already supports Array.prototype.includes
https://babeljs.io/docs/usage/polyfill/

> ... This means you can use new built-ins like Promise or WeakMap, static methods like Array.from or Object.assign, instance methods like Array.prototype.includes, ...